ctk/common/src/main/kotlin/org/codice/compliance/utils/decorators/IdpRedirectResponseDecorator.kt

```diff
@@ -50,17 +50,6 @@ internal constructor(response: IdpRedirectResponse) : IdpRedirectResponse(respon
         paramMap[SIGNATURE]
     }
 
-    val extraUrlParameters: Boolean by lazy {
```

Comment:
Removed this unused method.

Reply:
I'd say even if we keep it, there are better ways of determining its value.